package.json

```diff
@@ -22,6 +22,7 @@
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-icons": "^5.0.1",
```

Comment:
i installed the ant-design icon library and I'm curious if we need to download another icon library